### DISCUSSION TITLE
error : "Interaction already replied"

### DISCUSSION THREAD

**Discussion by AbdulKhaderWali** [imranbarbhuiya/pagination.djs#140] [Q&A] [answered]
I am building a pagination, using the example code already given in documentation , but getting this error
![Screenshot 2023-04-22 200310](https://user-images.githubusercontent.com/105049652/233790962-4a03a4a8-7c21-48fb-b147-2801c10cff45.png)

my code is here : https://github.com/AbdulKhaderWali/practice-bot

starting at line 150 in index.js

**Comment by imranbarbhuiya** [comment, marked as answer]
This isn't coming from this package but your code at line 173. U forgot to add `break` on last switch case
